tests/unit/localService.test.js

```diff
@@ -14,33 +14,61 @@ var fileSystemRepositoryStub = {};
 var workspaceRepositoryStub = {};
 
 var fileList = [
+  // [0] Unchanged file
   'api1.raml',
+  // [1] Unchanged file // Deleted remotely
   'api2.raml',
+  // [2] Changed locally // Changed remotely
   'api3.raml',
+  // [3] Changed locally // Deleted remotely
   'api4.raml',
+  // [4] Changed locally // Deleted remotely
+  'api5.raml',
+  // [5] Added locally
   'schema.json',
+  // [6] Changed locally
   'another.json',
+  // [7] Added locally // Already exists remotely
```

Comment:
As contentGenerator returns apiN.raml files for workspace, I think comments are ok